src/components/profile/events.js

```diff
@@ -1,32 +1,57 @@
-import React, { Component } from "react";
+import React from "react";
+import PropTypes from "prop-types";
 import EventsForProfile from "../event/EventsForProfile/EventsForProfile";
 import "moment-timezone";
 import "./User-profile.css";
 
-export default class Events extends Component {
-  render() {
-    const { data } = this.props.events;
+const Events = ({ events, currentUser, typeOfEvents }) => {
+  const { data } = events;
```

Comment:
What does this data objects mean. I'm not able to see this one in proptypes. Maybe it should be on same level as events prop

Reply:
> What does this data objects mean. I'm not able to see this one in proptypes. Maybe it should be on same level as events prop

I am not sure that it will be correct to make it on the same level, as we set our data that exists only in events props, and props  pass only events.